pkg/logs/input/eventlog/tailer.go

```diff
@@ -0,0 +1,81 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2018 Datadog, Inc.
+
+package eventlog
+
+import (
+	"fmt"
+
+	"github.com/DataDog/datadog-agent/pkg/logs/config"
+	"github.com/DataDog/datadog-agent/pkg/logs/message"
+	log "github.com/cihub/seelog"
+	"github.com/clbanning/mxj"
```

Comment:
Would `encoding/xml` do the same job here ?

Reply:
From the documentation:
>Unmarshal parses the XML-encoded data and stores the result in the value pointed to by v, which must be an arbitrary struct, slice, or string. Well-formed data that does not fit into v is discarded.

I was under the impression that it was not possible to unmarshal into `map[string]interface{}` values which is what we're looking for and what mxj does

Reply:
thanks for pining out the documentation